src/lavinmq/queue/queue.cr

```diff
@@ -32,6 +32,7 @@ module LavinMQ
     @consumers = Array(Client::Channel::Consumer).new
     @consumers_lock = Mutex.new
     @message_ttl_change = Channel(Nil).new
+    @basic_get_unacked = Deque(Envelope).new
```

Comment:
why storing the whole envelope? Only need the `SegmentPosition`, `Channel` and `delivery_tag`, no?

Reply:
Yeah, was thinking we might need some info from the msg, but since we won't then we don't!